app/utils/classifications_uploaded_utils.py

```diff
@@ -0,0 +1,47 @@
+from config import Configuration
+from werkzeug.utils import secure_filename
+import os
+
+
+def check_extensions(filename):
+    '''
+    this function checks if the extension of the file uploaded is included
+    among those specified in the tuple allowed_extension
+    if it is return True
+    else return False
+    '''
+    if len(filename.split('.')) == 2:
```

Comment:
Why this check? is it necessary?

Reply:
Since the "secure_filename" function provided by the" werkzeug.utils" package does not sanitize filenames with multiple extensions, such as 'image.php.png', it is necessary to implement additional checks beyond the "secure_filename" function in order to prevent malicious uploads on the web server. Although this check was not required, it was implemented to prioritize web security over functionality.
 
### Below is shown the output of the above mentioned string given by the "secure_filename" function:

![immagine_2023-06-19_151807380](https://github.com/unica-isde/flask-classification-2022-G/assets/91947876/2d3e600b-c0f4-450b-bc7e-c7e94629d407)